misc/cancel.talon

```diff
@@ -0,0 +1,6 @@
+# the actual behavior of "cancel cancel" is implemented in code/cancel.py
+# it allows you to prevent a command from executing by ending it with "cancel cancel"
```

Comment:
Maybe a quick NB that if they want to use a different phrase they also need to change it in the Python file? Alternatively could unify to single source of truth with a capture

Reply:
Added a note